tsdb/db_test.go

```diff
@@ -6008,10 +6040,20 @@ func testChunkQuerierOOOQuery(t *testing.T,
 				it := chunk.Chunk.Iterator(nil)
 				smpls, err := storage.ExpandSamples(it, newSample)
 				require.NoError(t, err)
+
+				// Verify that no sample is outside the chunk's time range.
+				for _, s := range smpls {
```

Comment:
If we know the samples we appended, do you think it would be worth also checking that chunk.MinTime and chunk.MaxTime equal the oldest and newest timestamp?

Reply:
added, thanks!